app/src/test/java/gateway/ITFileIO.java

```diff
@@ -90,7 +92,43 @@ static class State
 		assertEquals (500, CtrlFileUpload.file_upload (args).code, "Can't reach Auth");
 	}
 
-	@Test @Order (2) void file_check () throws InterruptedException
+	@Test @Order (3) void uploadFileOnDirectory ()
```

Comment:
In this test there is not code of error? Like 400, 409...? 🤔

Reply:
This test exclusively tests the case when a file is uploaded in a directory.

Tests for the create directory endpoint are located in `ITFileManagment.java`.